tests/unit/test_is_close.py

```diff
@@ -0,0 +1,302 @@
+"""Test `is_close()`."""
+
+
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pytest
+from skyreader import SkyScanResult
+from skyreader.result import PyDictResult
+
+
+@pytest.fixture
+def json_diff(request: Any) -> Path:
+    """Use the tests name to create a json filename."""
+    return Path(request.node.name + ".json")
+
+
+def test_000(json_diff: Path) -> None:
```

Comment:
The test numbering is a bit obscure? Is this some kind of convention or a free choice?

Reply:
It's my convention :smile: 

I eventually settled on 3 digits from previous projects because it allows 10+ similar tests (where the first digit is the test "variety"). Using numbers also helps to find a test without using line numbers.